src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/client/VisitSchedulerClient.kt

```diff
@@ -474,6 +474,13 @@ class VisitSchedulerClient(
     .retrieve()
     .bodyToMono<List<VisitRequestSummaryDto>>().block(apiTimeout)
 
+  fun approveVisitRequestByReference(visitReference: String): VisitDto = webClient.put()
+    .uri("/visits/requests/$visitReference/approve")
+    .accept(MediaType.APPLICATION_JSON)
+    .retrieve()
+    .bodyToMono<VisitDto>()
+    .block(apiTimeout)!!
+
```

Comment:
should we return a null like we do in bookVisitSlot just in case we encounter a 400 (or similar) error